timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -379,6 +379,13 @@ def score_and_cache_oof(
         if store_val_score:
             self.val_score = self._score_with_predictions(val_data, self._oof_predictions)
 
+    def _update_hpo_train_fn_kwargs(self, train_fn_kwargs: dict) -> dict:
```

Comment:
Is it more natural to make this `_get_hpo_train_fn_kwargs` instead of `_update`?